### PR TITLE
move to istio 1.8

### PR DESCRIPTION
**Pull request by ycai-aspen**
this is to move to istio 1.8 release. Didn't change go version.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -26,18 +23,14 @@ require (
...
+	k8s.io/api v0.19.3
+	k8s.io/apimachinery v0.19.3
+	k8s.io/client-go v0.19.3
 )
 
 replace (
-	github.com/golang/protobuf => github.com/golang/protobuf v1.3.2
-	k8s.io/client-go => k8s.io/client-go v0.18.0
+	github.com/golang/protobuf => github.com/golang/protobuf v1.4.2
+	k8s.io/client-go => k8s.io/client-go v0.19.3
```
Comment (m-eaton) [inline go.mod:35]: if the versions stay after `make` is run, these can be removed

**Review by m-eaton — APPROVED** [review]
LGTM. If the protobuf and k8s versions do not change when we run a `go mod tidy` and `make`, we should be able to remove the replace section, but it will work either way

File: go.mod (hunk 1 above)
Comment (ycai-aspen) [reply go.mod:35]: that didn't work. 
../../../../pkg/mod/k8s.io/client-go@v0.18.0/kubernetes/scheme/register.go:27:2: module k8s.io/api@latest found (v0.20.1), but does not contain package k8s.io/api/auditregistration/v1alpha1

Comment (m-eaton) [reply go.mod:35]: sounds like a protobuf issue. what happens when you only remove line 35 with the k8s api?

**Pull request merged**